src/topic/components/Participants.js

```diff
@@ -1,17 +1,44 @@
 import React from 'react';
 import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import PieChart from 'react-svg-piechart';
+import Colors from '../../styles/Colors';
+
+const Participants = props => {
```

Comment:
A general rule of thumb is - if you are using any object more than once, destruct it! (you may want to take a look at fb's react's example [there](https://reactjs.org/blog/2015/12/18/react-components-elements-and-instances.html)

In `Participants` we have it twice, so we can improve it by declaring:
`const Participants = ({likes}) => { ... }`